src/schema/v1/artwork/index.ts

```diff
@@ -585,6 +585,20 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         description:
           "Is this work available for shipping only within the Contenental US?",
+        deprecationReason: deprecate({
+          inVersion: 2,
+          preferUsageOf: "onlyShipsDomestically",
+        }),
+        resolve: artwork => {
+          return Boolean(
+            artwork.domestic_shipping_fee_cents &&
+              artwork.international_shipping_fee_cents == null
+          )
+        },
+      },
+      onlyShipsDomestically: {
```

Comment:
I took a look at where this is used, and it highlights something else we'll have to do before launching to the UK--

https://github.com/artsy/reaction/blob/8faaae88e740ec99c61ae1990f590cc0726bb2fe/src/Apps/Order/Routes/Shipping/index.tsx#L338

Right now if `shipsToContinentalUSOnly` is true, we disable the country select dropdown on the shipping form and default it to "United States".

In order to make this country-agnostic, we'll have to update that logic to pin the country select to the country that's being shipped from. I made https://artsyproduct.atlassian.net/browse/PURCHASE-1476 to track!